src/subclass/object.rs

```diff
@@ -692,6 +692,9 @@ mod test {
             Some("return value".to_value())
         })
         .expect("Failed to connect on 'create-string'");
+        obj.connect_notify(Some("create-string"), |_, _| {
+            println!("hello!");
```

Comment:
This is not super useful :) Instead of printing something, change some `Arc<AtomicBool>` or something from `false` to `true` here and assert that afterwards

Reply:
Also this will never be called: notify is for property changes.

Reply:
Well, it crashed for me at least? (I used a pre-fix version)

Reply:
Well that's a bug then. This callback should never be called

Reply:
Oh my bad, I thought it was part of the test I added, this one shouldn't have been there. Removed!